tests/system/providers/google/cloud_sql/example_cloud_sql.py

```diff
@@ -280,51 +242,60 @@
 
     # [START howto_operator_cloudsql_replicas_delete]
     sql_instance_failover_replica_delete_task = CloudSQLDeleteInstanceOperator(
-        project_id=GCP_PROJECT_ID,
         instance=FAILOVER_REPLICA_NAME,
         task_id='sql_instance_failover_replica_delete_task',
     )
 
     sql_instance_read_replica_delete_task = CloudSQLDeleteInstanceOperator(
-        project_id=GCP_PROJECT_ID, instance=READ_REPLICA_NAME, task_id='sql_instance_read_replica_delete_task'
+        instance=READ_REPLICA_NAME, task_id='sql_instance_read_replica_delete_task'
     )
     # [END howto_operator_cloudsql_replicas_delete]
 
     # [START howto_operator_cloudsql_delete]
     sql_instance_delete_task = CloudSQLDeleteInstanceOperator(
-        project_id=GCP_PROJECT_ID, instance=INSTANCE_NAME, task_id='sql_instance_delete_task'
-    )
-    sql_instance_delete_task2 = CloudSQLDeleteInstanceOperator(
-        project_id=GCP_PROJECT_ID, instance=INSTANCE_NAME2, task_id='sql_instance_delete_task2'
+        instance=INSTANCE_NAME, task_id='sql_instance_delete_task'
     )
```

Comment:
Removal of instances should be also done with TriggerRule.ALL_DONE (in case test fail after creating an instance). 

To avoid showing it inside documentation you can add attribute later:
# [START  howto_smth]
    task = SmthOperator()
# [END  howto_smth]
task.trigger_ruke = TriggerRule.ALL_DONE